conf/messages.cy

```diff
@@ -95,16 +95,17 @@ in-person-contact.heading=Cysylltu â ni dros y ffôn neu gael ymweliad personol
 in-person-contact.text-relay=Rydym yn cynnig gwasanaeth ‘text relay’ os ydych yn fyddar, â nam ar eich clyw neu os oes gennych nam ar eich lleferydd.
 in-person-contact.bsl=Gallwn ddarparu dehonglydd Iaith Arwyddion Prydain (BSL), neu gallwch drefnu ymweliad gan ymgynghorydd CThEM i’ch helpu i gwblhau’r gwasanaeth.
 in-person-contact.find-out-how.part1='Dysgwch '
+in-person-contact.find-out-how.link=https://www.gov.uk/cael-help-cthem-cymorth-ychwanegol
 in-person-contact.find-out-how.part2=sut i gael cymorth ychwanegol oddi wrth CThEM
 
 ## Technical information
 technical.heading=Gwybodaeth dechnegol am hygyrchedd y gwasanaeth hwn
 technical.commitment=Mae CThEM wedi ymrwymo i wneud y gwasanaeth hwn yn hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau Symudol) (Rhif 2) 2018.
 technical.compliance.full='Mae’r gwasanaeth hwn yn cydymffurfio’n llawn â '
 technical.compliance.partial='Mae’r gwasanaeth hwn yn cydymffurfio’n rhannol '
```

Comment:
@JoPintoPaul just spotted that this one doesn't add an â after this message, do you think it's worth checking this with WLU? think this was requested in the original bug report, might be getting mixed up here though